### ISSUE TITLE
Use containerized Composer and WP-CLI

### ISSUE THREAD

**Issue by philipnewcomer** [Pilothouse-App/pilothouse#57]
Composer should be run inside a PHP container, instead of on the host. This will make the entire stack more portable, and eliminate issues with a package requiring a PHP version higher than what is currently installed on the host.

Will also help with cross-platform compatibility by not requiring PHP to be installed on Windows and Linux hosts where it is not available by default.

WP-CLI should be actually installed and managed in/by the container instead of being copied from the host where it is managed via Composer.

**Comment by philipnewcomer** [comment]
Updated in develop branch as this is a breaking change

**Comment by AaronHolbrook** [comment]
What do you anticipate this breaking?

**Comment by philipnewcomer** [comment]
It's breaking because there are still some edge cases that need to be worked out with Composer. However, I realized that since I implemented the containerized Composer command as `pilothouse composer [...]` so that we can still use Composer on the host to work around those edge cases, It's not really breaking under normal use. So I've committed this to master.

The containerized Composer command is (for now) `pilothouse composer [...]` instead of just `composer [...]`. There are some edge cases with ssh keys that are used to access private GitHub repos via Composer not being available in the PHP Composer. In those cases GitHub/Composer asks you to create an access token, which should work, but I'd like to get it to where it's completely seamless. I'll need to find a workaround for that, ideally syncing the user's `.ssh` directory into the PHP container, but there were some blockers I experienced when attempting that. So in the meantime, until I can get those figured out, the containerized Composer is run using `pilothouse composer [...]` so that Composer on the host can still be run to resolve any issues if need be.

Once those edge cases are worked out, the containerized Composer command will be moved back to just `composer [...]` in the 1.0 release.
